mapflow/functional/layer_utils.py

```diff
@@ -361,6 +361,20 @@ def download_results_file(self, pid) -> None:
             timeout=300
         )
 
+    def download_aoi_file(self, pid) -> None:
+        """
+        Download area of interest and save to a geojson file
+        """ 
+        path = self.temp_dir+str(pid)+'_aoi.geojson'                             
+        self.dlg.saveResultsButton.setEnabled(False)
+        self.http.get(
+            url=f'{self.server}/processings/{pid}/aois',
+            callback=self.download_aoi_file_callback,
+            callback_kwargs={'path': path},
+            use_default_error_handler=True,
+            timeout=300
```

Comment:
300 seconds may be a bit long for such request, AOIs are ususally much smaller in bytes than the results are.

Let's try 30 seconds for now, and we'll test it on the biggest user's AOIs to check if it is ok.